### DISCUSSION TITLE
About the instructions

### DISCUSSION THREAD

**Discussion by sukhman-shergill** [mishl-dev/Discord-AI-Chatbot#177] [General]
Right now, I have to always change the instructions in config to assist or anything else by going to the panel. Isn't there any command through which I can change the value of that from discord by user input or something?? 
Please tell.

**Comment by mishl-dev** [comment]
no

**Reply by sukhman-shergill** [reply]
can I make the bot able to send files, pics or links??

**Reply by mishl-dev** [reply]
yea

**Reply by mishl-dev** [reply]
Bot cant send file and pics in normal convo yet but you can do it yourself cus the code is public

**Reply by sukhman-shergill** [reply]
ok thnx
